exporter/opencensusexporter/factory.go

```diff
@@ -168,15 +169,17 @@ func (f *Factory) CreateMetricsExporter(logger *zap.Logger, config configmodels.
 	if err != nil {
 		return nil, nil, err
 	}
+	// TODO(ccaraman) Fix the exporter name set. It is expected to us the full name <exporter type>(/<name from config>
```

Comment:
Please, create an issue and link it here like you did below.

Reply:
https://github.com/open-telemetry/opentelemetry-service/issues/265 i'll update the comment.